CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed an issue that caused `UnrealGDK/Setup.sh` to report `sed: can't read : No such file or directory` when run on macOS.
 - Static subobjects on bNetLoadOnClient actors are now removed on clients in a manner matching native unreal's behavior. This change affects subobjects removed by the server while the actor is not in the client's interest.
 - Fixed an issue where multicast rpcs could be overwritten and then dropped on authority flicker.
+- Fixed issue using squid snapshot end point with a local deployment, using `localhost:5006/snapshot` works again and creates a snapshot.
```

Comment:
Squid is not publicly known. Please just refer to as runtime.

Reply:
ok didn't know that.